enums/markerStatus.js

```diff
@@ -0,0 +1,5 @@
+export default {
+  TODO: 'TODO',
```

Comment:
what does this markerStatus enum do? 
Maybe it makes sense to make an object with:  
`export default {
  TODO: {
     name: 'TODO',
     color: 'red'
  },
}`

then we can get rid of computed color and use it elsewhere. What do you think?

Reply:
i left the enum but also added the status color map, so we don't need to compute the color. thanks!